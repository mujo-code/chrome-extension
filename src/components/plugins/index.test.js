import React from 'react'
import { create } from 'react-test-renderer'
import { Plugins } from '.'

jest.mock('../../plugins')
// eslint-disable-next-line import-order-alphabetical/order
const { getPlugins } = require('../../plugins')

test('Tabs component matches snapshot', () => {
  console.error = jest.fn()
  getPlugins.mockReturnValue([
    function Foo() {
      return <>Foo</>
    },
    function Bar() {
      throw new Error('bar')
    },
  ])
  const tree = create(<Plugins />).toJSON()
  expect(tree).toMatchSnapshot()
})
