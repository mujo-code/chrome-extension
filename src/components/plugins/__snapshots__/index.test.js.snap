// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tabs component matches snapshot 1`] = `"Foo"`;
